app/views/edit/deployment-config.html

```diff
@@ -0,0 +1,373 @@
+                                  <span class="help-block">
+                                    Must be a non negative whole number or percentage.
+                                  </span>
+                                </div>
+                              </div>
+                            </div>
+
+                            <div ng-if="view.advancedStrategyOptions">
+                              <div class="lifecycle-hooks">
+                                <div class="lifecycle-hook" id="pre-lifecycle-hook">
+
+                                  <h3>Pre Lifecycle Hook</h3>
+                                  <lifecycle-hook model='strategyData[strategyParamsPropertyName].pre'
+                                                    type='pre'
+                                                    available-volumes='volumeNames'
+                                                    available-containers='containerNames'
+                                                    namespace="projectName">
+                                  </lifecycle-hook>
+                                </div>
+
+                                <div ng-if="strategyData.type !== 'Rolling'" class="lifecycle-hook" id="mid-lifecycle-hook">
+                                  <h3>Mid Lifecycle Hook</h3>
+                                  <lifecycle-hook model='strategyData[strategyParamsPropertyName].mid'
+                                                    type='mid'
+                                                    available-volumes='volumeNames'
+                                                    available-containers='containerNames'
+                                                    namespace="projectName">
+                                  </lifecycle-hook>
+                                </div>
+
+                                <div class="lifecycle-hook" id="post-lifecycle-hook">
+                                  <h3>Post Lifecycle Hook</h3>
+                                  <lifecycle-hook model='strategyData[strategyParamsPropertyName].post'
+                                                    type='post'
+                                                    available-volumes='volumeNames'
+                                                    available-containers='containerNames'
+                                                    namespace="projectName">
+                                  </lifecycle-hook>
+                                </div>
+                              </div>
+                            </div>
+                          </div>
+                          <div class="gutter-top">
+                            <div ng-if="!view.advancedStrategyOptions">To set additional parameters or edit lifecycle hooks, view <a href="" ng-click="view.advancedStrategyOptions = true">advanced strategy options</a></div>
```

Comment:
Here I'd have the period after the link since the entire sentence is not an action link, but I'd leave the period off for the hide link.